Updates

steviebeenz opened the pull request:
!wip

Machine-Maker commented on src/main/frontend/src/components/entrypoints/users/Notifications.vue:
```diff
@@ -159,8 +159,8 @@ export default {
-        const invites = $('.invite-content');
-        invites.css('height', invites.width());
+        const invites = document.querySelector('.invite-content');
+        invites.style.height = invites.style.width;
```
this is **not** the same thing. The element has no width. The jquery width method calculates the actual width of the element. What you changed it to is what I originally had, but it's not the same, and won't work.

steviebeenz replied:
Thats from before testing. Will work our another way of doing this WITHOUT JQUERY

steviebeenz replied:
It is .offsetWidth for reference.

Machine-Maker commented on src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue:
```diff
@@ -370,7 +370,7 @@ export default {
             alert.fadeIn('slow');
 
-            $('#form-url-upload').css('display', 'none');
+            document.querySelector('form-url-upload').style.display = "none";
```
I think this needs to have a `#` before it right?

steviebeenz replied:
No it needs to be a getElementById. Thanks though

steviebeenz commented:
ZappFix Head Dump

      console.error("ZAPPFIX: Fixed Empty Property (Array) (REMOVE WHEN FIXED)");
      window.COLORS = [];
      console.error("ZAPPFIX: Fixed Empty Property (STRING) (REMOVE WHEN FIXED)");
      window.DiscourseEmbed = {};
      window.DiscourseEmbed.discourseUrl = ""
      console.error("ZAPPFIX: Fixed Empty Property (Array) (REMOVE WHEN FIXED)");
      window.CHANNELS = []
      console.error("ZAPPFIX: Fixed Empty Property (OBJECT) (REMOVE WHEN FIXED)");
      window.csrfInfo = {}
      console.error("ZAPPFIX: Fixed Empty Property (STRING) (REMOVE WHEN FIXED)");
      window.csrfInfo.headerName = ""

steviebeenz commented:
@Machine-Maker It's all resolved (I hope). Thanks for the suggestions.

steviebeenz commented:
Ok so here I will go into depth about what this PR is. It is some Jquery Removal, and some compression (+ ZAPPFIX to temporarily patch some bugs) .

MiniDigger commented on src/main/frontend/.postcssrc.js:
```diff
@@ -1,5 +1,6 @@
 module.exports = {
     plugins: {
         autoprefixer: true,
+        cssnano: true
```
our css is already minimized, remove this

MiniDigger commented on src/main/frontend/public/index.html:
```diff
@@ -6,6 +6,21 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <!-- START ZAPPFIX BETA INJECT -->
```
wtf is this?

MiniDigger commented on src/main/frontend/vue.config.js:
```diff
@@ -18,6 +19,8 @@ module.exports = {
     configureWebpack: {
         plugins: [new StatsPlugin('stats.json')],
         optimization: {
+            minimize: true,
```
our js is already minimzed

MiniDigger reviewed (changes requested):
yeah, Idk abotu this

kashike commented:
Yea, I am going to close this pull request. Please actually look at what is already implemented before working on something, and keep the irrelevant and useless changes out of pull requests.
